src/main/g8/default.properties

```diff
@@ -1,6 +1,7 @@
 description = This template generates a Play Scala project
 name = play-scala-seed
 organization=com.example
+package = $organization$.$name;format="lower,word"$
```

Comment:
It's kind of weird that this defaults to a hyphenated value `com.example.play-scala-seed`. Should we change the name, or maybe use a different value for the package?

Reply:
Ok nevermind. This is "lower,word" which removes non-word chars. So it would be `com.example.playscalaseed`.